tests/console/zypper_ar.pm

```diff
@@ -19,7 +19,7 @@ sub run {
     select_console 'root-console';
     # Trying to switch to more scalable solution with updated rsync.pl
     if (my $urlprefix = get_var('MIRROR_PREFIX')) {
-        my @repos_to_add = qw(OSS NON_OSS OSS_DEBUGINFO LEAP_MICRO);
+        my @repos_to_add = qw(OSS NON_OSS OSS_DEBUGINFO LEAP_MICRO SLE_LEAP_16_0_AARCH64_PPC64LE_S390X_X86_64);
```

Comment:
This is clearly wrong here

Reply:
Should we move ahead and have https://github.com/os-autoinst/openqa-trigger-from-obs/pull/294 merged and deployed for now?

My question to Andrii is still open https://github.com/os-autoinst/openqa-trigger-from-obs/pull/294#discussion_r2088984222 but given we're headed for labs next week,